src/js/jwt/insights/user.js

```diff
@@ -63,7 +66,10 @@ module.exports = (token) => {
                 }
             }
 
-            return user;
+            return {
```

Comment:
@ryelo @karelhala 
I think we actually want
```javascript
user.identity.entitlements = data;
return user;
```
Ryan the 3scale object is going to look like this, right?
```javascript
{
  "identity": {
    "account_number": "6089719",
    "type": "User",
    "user": {
      "username": "rhn-insightsqe-ihands",
      "email": "ihands+qa@redhat.com",
      "first_name": "Ian",
      "last_name": "Hands",
      "is_active": true,
      "is_org_admin": true,
      "is_internal": false
    },
    "internal": {
      "org_id": 52399467
    },
    "entitlements": {
      "hybrid_cloud": {
        "is_entitled": true
      },
      "insights": {
        "is_entitled": true
      },
      "openshift": {
        "is_entitled": true
      },
      "smart_management": {
        "is_entitled": false
      }
    }
  }
}
```

Reply:
I didn't wanted to mess with identity object, but if 3scale's gonna return it in this way I'll update it. Thanks for the review and test!

Reply:
The entitlements stuff isn't going to return "entitlements"
just:
```
      "hybrid_cloud": {
        "is_entitled": true
      },
      "insights": {
        "is_entitled": true
      },
      "openshift": {
        "is_entitled": true
      },
      "smart_management": {
        "is_entitled": false
      }

```

Reply:
No, what we are wondering here is what the shape of the x-rh-identity header object is.
3scale adds the entitlements response to the identity object.... what does it look like after that?

@coderbydesign what does the object look like after you are done mixing in entitlements?
```javascript
{
  identity: ...
  entitlements: ...
}
```

Or
```javascript
identity: {
  entitlements: ...
  ...
}
```

?? Or something different

Reply:
@iphands what I'm working on right now has `entitlements` nested under `identity`, as the second example shows. 

Per your comment, @ryelo, I'm expecting the entitlements payload to come from the service as you mention, not nested under an `entitlements` key, but I'm setting entitlements on the identity header explicitly, something like:
```
identity.entitlements = entitlements_response_body
```
so that it does come back in `x-rh-identity` as:
```json
{
  "identity": {
    "account_number": "6089719",
    "type": "User",
    "user": {
      "username": "rhn-insightsqe-ihands",
      "email": "ihands+qa@redhat.com",
      "first_name": "Ian",
      "last_name": "Hands",
      "is_active": true,
      "is_org_admin": true,
      "is_internal": false
    },
    "internal": {
      "org_id": 52399467
    },
    "entitlements": {
      "hybrid_cloud": {
        "is_entitled": true
      },
      "insights": {
        "is_entitled": true
      },
      "openshift": {
        "is_entitled": true
      },
      "smart_management": {
        "is_entitled": false
      }
    }
  }
}
```

Reply:
@karelhala so I think the current code matches ^ w/r/t the entitlements piece.
Merging